### ISSUE TITLE
Invalid inputs should have aria-invalid attribute

### ISSUE THREAD

**Issue by twgraham** [reactstrap/reactstrap#1781]
- components: `Input`, `CustomInput`, `CustomFileInput`
- reactstrap version `#8.4.1`
- import method `es`
- react version `#16.12.0`
- bootstrap version `#4.3.1`

### What is happening?
Input's that are invalid (i.e. have the invalid prop set to true), do not have the aria-invalid attribute applied.

### What should be happening?

Invalid input's have aria-invalid attribute.

### Steps to reproduce issue

1. <Input invalid={true} {...otherProps} />
2. The rendered DOM node doesn't have aria-invalid

**Comment by R-Oscar** [comment]
Seems like this is fixed, the issue should be closed then? :)
